src/script/chem/molfile/molfile.js

```diff
@@ -289,6 +290,26 @@ Molfile.prototype.writeCTab3000 = function (molecule) { // eslint-disable-line m
 	this.writeCR('M  V30 END CTAB');
 };
 
+Molfile.prototype.writeRGroups3000 = function (molecule) { // eslint-disable-line max-statements
```

Comment:
No JSDoc?

Reply:
JSDoc is on some functions but not others in this file. Is this intentional?

Reply:
Added.
Nope it's not intentional.. The existing V2000 writer methods are not documented, so we should be adding documentation to the new V3000 ones that we're adding, at the least.